src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -121,24 +121,25 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
       return;
     }
 
-    this._unsubRenderTemplate = subscribeRenderTemplate(
-      this.hass.connection,
-      (result) => {
-        this._content = result;
-      },
-      {
-        template: this._config.content,
-        entity_ids: this._config.entity_id,
-        variables: {
-          config: this._config,
-          user: this.hass.user!.name,
+    try {
+      this._unsubRenderTemplate = await subscribeRenderTemplate(
+        this.hass.connection,
+        (result) => {
+          this._content = result;
         },
-      }
-    );
-    this._unsubRenderTemplate.catch(() => {
+        {
+          template: this._config.content,
+          entity_ids: this._config.entity_id,
+          variables: {
+            config: this._config,
+            user: this.hass.user!.name,
+          },
+        }
+      );
+    } catch {
```

Comment:
This breaks ES5 build. Optional catch binding is pretty new. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#Browser_compatibility

![image](https://user-images.githubusercontent.com/1444314/82584289-268e4f80-9b49-11ea-83a7-c086eeba9362.png)